x/farming/types/proposal.go

```diff
@@ -43,65 +40,18 @@ func (p *PublicPlanProposal) ProposalRoute() string { return RouterKey }
 func (p *PublicPlanProposal) ProposalType() string { return ProposalTypePublicPlan }
 
 func (p *PublicPlanProposal) ValidateBasic() error {
```

Comment:
We need detailed validation logic on here for checking validation each `AddRequestProposal`, `AddRequestProposal`, `DeleteRequestProposal` types with Validate* functions and Validation PublicPlans for example) total ratio verification of plans with same `FarmingPoolAddress`